apps/dashboard/src/hooks/use-feature-flag.tsx

```diff
@@ -0,0 +1,23 @@
+import { useFlags } from 'launchdarkly-react-client-sdk';
+import { FeatureFlagsKeysEnum, prepareBooleanStringFeatureFlag } from '@novu/shared';
+import { LAUNCH_DARKLY_CLIENT_SIDE_ID } from '../config';
+
+function isLaunchDarklyEnabled() {
+  return !!LAUNCH_DARKLY_CLIENT_SIDE_ID;
+}
+
+export const useFeatureFlag = (key: FeatureFlagsKeysEnum, defaultValue = false): boolean => {
+  const flags = useFlags();
+
+  if (!isLaunchDarklyEnabled()) {
+    const envValue =
+      // Check window._env_ first
+      window?._env_?.[key] ??
```

Comment:
I don't think we have the `_env_` fallback on the window object in the new Dashboard. This was in the old web app. Instead, we should rely on VITE_ prefixed env variables as a fallback.